libqtile/backend/x11/xcore.py

```diff
@@ -206,7 +206,7 @@ def remove_listener(self) -> None:
     def _remove_listener(self) -> None:
         if self.fd is not None:
             logger.debug("Removing io watch")
-            loop = asyncio.get_event_loop()
+            loop = asyncio.get_running_loop()
```

Comment:
I guess this is just following along with the changes in the asyncio API?

Reply:
https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.get_event_loop

yeah, get_running_loop is preferred over get_event_loop

Reply:
Great, thanks.